composer.json

```diff
@@ -17,7 +17,7 @@
         "react/event-loop": "^1.0 || ^0.5",
         "react/stream": "^1.0 || ^0.7 || ^0.6 || ^0.5 || ^0.4",
         "react/socket": "^1.0 || ^0.8 || ^0.7",
-        "react/dns": "~0.4.0",
+        "react/dns": "~0.4.13",
```

Comment:
Might as well target [`^1.2 || ^0.4.13`](https://github.com/reactphp/dns/releases) since `react/dns` 1.0 has been out for a while

Reply:
I'll make this edit tonight.

Reply:
I made this chance and tests failed in Travis-CI and I have no idea what the error messages mean. Can someone help?

Reply:
After the failure in commit 90fe2c0 I think adopting `react/dns` 1.x is out of scope for this pull request.